README.md

```diff
@@ -1,2 +1,22 @@
-# dev-resources-kit
-hello world test1
+# Holochain Developer Resource Kit
+## Summary
+This repo is meant to capture all the community contributed and sponsored content related to
+development in HoloChain. It is meant to serve as a resource for new beginner to intermediate level
+developers that want to learn more about the Holochain technology and how to contribute to the
+ecosystem.
+
+## Goals
+* Provide an understanding the architecture and features of Holochain and design patterns to consider
+  when building a Holochain application (hApp);
+* Common setup and toolchain guidance for developers of all levels that has been tested by the community and \n
+  is known to work;
+* Tutorial projects that allow step-by-step understanding of how to build a hApp and contribute to either hApps \n
+  or the core Holochain project;
+* Example applications that explore all the aspects of the Holochain technology and serve a useful purpose for \n
+  the community and stated education objectives;
+
+## Resources
+* See [Awesome-Holochain](awesome.md)
+* See [Holochain Developer Portal](https://dev.holochain.org)
```

Comment:
https://developer.holochain.org :)

Reply:
👍 got it. besides that edit - any problems merging this upstream @Connoropolous @dhtnetwork

Reply:
> 👍 got it. besides that edit - any problems merging this upstream @Connoropolous @dhtnetwork

My only request. Can we consider changing the name to holochaindoc? Global name change request. We want people to be clear that it's Holochain and not  @Holo.  
projects/holochaindoc/specs/index.md updated from

Reply:
Agreed, that'd be better